beekeeper-path-scheduler-apiary/src/test/java/com/expediagroup/beekeeper/scheduler/filter/WhitelistedListenerEventFilterTest.java

```diff
@@ -47,17 +47,17 @@
   @EnumSource(value = EventType.class, names = { "ALTER_PARTITION",
                                                  "ALTER_TABLE" })
   public void filterDefaultEvents(EventType eventType) {
-    when(listenerEvent.getEventType()).thenReturn(eventType);
-    boolean filter = listenerEventFilter.filter(listenerEvent, UNREFERENCED);
+    when(this.listenerEvent.getEventType()).thenReturn(eventType);
```

Comment:
And here, is this Intellij doing formatting?

Reply:
Make it sttttttop! ;P